inc/Admin.php

```diff
@@ -216,35 +228,51 @@ private function should_show_welcome_notice(): bool {
 	}
 
 	/**
-	 * Get the Otter Blocks plugin status.
+	 * Get the WP Full Pay plugin status.
 	 *
 	 * @return string
 	 */
-	private function get_otter_status(): string {
+	private function get_wpfp_status(): string {
 		$status = 'not-installed';
 
-		if ( is_plugin_active( 'otter-blocks/otter-blocks.php' ) ) {
+		if ( is_plugin_active( 'wp-full-stripe-free/wp-full-stripe.php' ) ) {
 			return 'active';
 		}
 
-		if ( file_exists( ABSPATH . 'wp-content/plugins/otter-blocks/otter-blocks.php' ) ) {
+		if ( file_exists( ABSPATH . 'wp-content/plugins/wp-full-stripe-free/wp-full-stripe.php' ) ) {
 			return 'installed';
 		}
 
 		return $status;
 	}
 
 	/**
-	 * Update Otter reference key.
+	 * Run after WP Full Pay activation.
+	 *
+	 * @param string $plugin Plugin name.
+	 *
+	 * @return void
+	 */
+	public function after_wpfs_activation( $plugin ) {
+		if ( 'wp-full-stripe-free/wp-full-stripe.php' !== $plugin ) {
+			return;
+		}
+
+		update_option( Constants::CACHE_KEYS['dismissed-welcome-notice'], 'yes' );
+		exit;
+	}
```

Comment:
Just a small suggestion, I think you can use this hook for the activation: https://developer.wordpress.org/reference/functions/register_activation_hook/

This will remove the need for the `if` and `exit`.

Reply:
Usually that hook is used by plugins to fire things when they get activated itself, while this is a catch-all incase we add more effects for other plugins.